packages/web/hooks/use-estimate-tx-fees.ts

```diff
@@ -91,6 +135,7 @@ export function useEstimateTxFees({
         accountStore,
```

Comment:
NIT: on line 130 could we change the query key to be `"estimate-tx-fees"` instead of `"simulate-swap-tx"`